packages/twenty-front/src/modules/ui/field/display/components/EllipsisDisplay.tsx

```diff
@@ -1,6 +1,8 @@
 import styled from '@emotion/styled';
 
 const StyledEllipsisDisplay = styled.div<{ maxWidth?: number }>`
+  align-items: center;
+  display: flex;
```

Comment:
With this we would be affecting all places where `EllipsisDisplay` is used, I am not sure this is what we want. Could you instead:
- add a `className` prop to `EllipsisDisplay` (so we can style it with Emotion)
- in the `CurrencyDisplay.tsx` file, style `EllipsisDisplay` locally with Emotion, like this:
```typescript
const StyledEllipsisDisplay = styled(EllipsisDisplay)`
  align-items: center;
  display: flex;
`
```